docs/source/getting_started/connecting.rst

```diff
@@ -0,0 +1,18 @@
+======================================
+Connecting to the Citrine Platform
+======================================
+
+The Citrine Python Client connects to the Citrine Platform API.
+To create a client, you must specify the specific site that you want to connect to and your API key:
+
+.. code-block:: python
+
+    from citrine import Citrine
+    import os
+    citrine_client = Citrine(host="stage.citrine-platform.com", api_key=os.environ.get("CITRINE_API_KEY")
```

Comment:
In addition to setting the host and API key, I has to set port and and api scheme environment variables before I was able to connect. Per the confluence instruction:
export CITRINE_API_PORT=443
export CITRINE_API_SCHEME=https
I haven't seen either of these set using the Python interface.

Reply:
Do you have to?  Those have default values here, so I don't think they need to be set: https://github.com/CitrineInformatics/citrine-python/blob/develop/src/citrine/citrine.py#L14

Reply:
I can't replicate the original errors I remember getting, so I guess you don't.